gloo/docs/enterprise/authentication/jwt.md

```diff
@@ -68,7 +69,8 @@ Let's see the claims for `svc-a` - the service account we just created:
 
 ```shell
 CLAIMS=$(kubectl exec test-pod cat /var/run/secrets/kubernetes.io/serviceaccount/token | cut -d. -f2)
-PADDING=$(head -c $((${#CLAIMS} % 3)) /dev/zero |tr '\0' =)
+PADDING_LEN=$(( (  4 - ( ${#CLAIMS} % 4 )  ) % 4 ))
+PADDING=$(head -c $PADDING_LEN /dev/zero |tr '\0' =)
 PADDED_CLAIMS="${CLAIMS}${PADDING}"
 echo $PADDED_CLAIMS | base64 --decode |jq .
```

Comment:
you've got some secret tabs in here, iterm2 warns when pasting. Can you replace with spaces?